docs/assets/themes/zeppelin/js/docs.js

```diff
@@ -107,13 +107,18 @@ function maybeScrollToHash() {
 
 $(function() {
   codeTabs();
-  viewSolution();
+  // Display anchor links when hovering over headers. For documentation of the
+  // configuration options, see the AnchorJS documentation.
+  anchors.options = {
+    placement: 'right'
```

Comment:
@AhyoungRyu Thanks for the good feature can promote the references! This the feature that I needed :)
But If you don't mind can you consider to change placement to left side instead of right side?
Because considering the user experiences of anchors in other famous open source project documentation:

- most of anchors tend to added at left side of header titles.
- users are tend to start drag or read document from left to right so left anchor have more chance to be discovered fast.
- It seems more clean and neat than right sided anchor.

https://github.com/bryanbraun/anchorjs
http://getbootstrap.com/getting-started/